DEVX-4960 Update Enable Audio Guide (#3469)
Nexmo/nexmo-developer: _partials/stitch/guides/enable-audio/kotlin.md

> You will now need to add two buttons for the user to enable and disable audio. Open `app/src/main/res/layout/fragment_chat.xml` file and add two new buttons (`enableMediaButton` and `disableMediaButton`) right below `sendMessageButton`. 

Reviewer: ```suggestion
Add two buttons for the user to enable and disable audio. Open the `app/src/main/res/layout/fragment_chat.xml` file and add two new buttons (`enableMediaButton` and `disableMediaButton`) immediately below the `sendMessageButton`. 
```

            app:layout_constraintBottom_toTopOf="@id/conversationEventsScrollView"
            app:layout_constraintLeft_toLeftOf="parent"
            app:layout_constraintTop_toTopOf="parent"
            android:text="Enable Audio" />

    <Button
            android:id="@+id/disableMediaButton"
            android:layout_width="wrap_content"
            android:layout_height="wrap_content"
            app:layout_constraintBottom_toTopOf="@id/conversationEventsScrollView"
            app:layout_constraintLeft_toLeftOf="parent"
            app:layout_constraintTop_toTopOf="parent"
            android:visibility="gone"
            android:text="Disable Audio"
            tools:visibility="visible"/>
<!--...-->
```

```kotlin
private lateinit var enableMediaButton: Button
private lateinit var disableMediaButton: Button
```

Retrieve the buttons references by adding `findViewById` calls in the `onViewCreated` method:

```java
public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {

    //...
    enableMediaButton = view.findViewById(R.id.enableMediaButton)
    disableMediaButton = view.findViewById(R.id.disableMediaButton)
}
```

> Add listeners to the buttons inside `onViewCreated` method:

Reviewer: ```suggestion
Add click event listeners for the buttons, inside the `onViewCreated` method:
```
